templates/telemetry/prio/prio.4.schema.json

```
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "prio",
+  "properties": {
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
+    "payload": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 1
+        },
+        "reason": {
+          "type": "string",
+          "enum": [
+            "shutdown",
+            "periodic",
+            "max"
+            ]
+        },
+        "prioData": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "encoding": {
+                "type": "string"
+              },
+              "prio": {
+                "type": "object"
```

Do you have the appropriate type for this object? It looks like it could be:
```suggestion
                "type": "object",
                "properties": {
                  "a": {
                    "type": "string"
                  },
                  "b": {
                    "type": "string"
                  }
                },
                "required": ["a", "b"]
```

---

For now it could be, but I was being specifically vague in the Proposal to cover my bases.